build.gradle

```diff
@@ -165,22 +165,24 @@ clean {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:3.2.0')
+    api platform('com.exactpro.th2:bom:4.0.3')
 
-    implementation group: 'com.exactpro.th2', name: 'common', version: '3.40.0-TH2-3789-2817159492-SNAPSHOT'
-    implementation group: 'com.exactpro.th2', name: 'common-utils', version: '0.0.1-dev-2427721512-SNAPSHOT'
+    implementation group: 'com.exactpro.th2', name: 'common', version: '3.44.0-TH2-3789-vulnerability-fix-3929224173-06f5e07-SNAPSHOT'
```

Comment:
Use release version

Reply:
It seems that special build is required here: https://github.com/th2-net/th2-common-j/commits/TH2-3789